frontend/app/src/components/table-filter/TableFilter.vue

```diff
@@ -372,6 +410,9 @@ const { t } = useI18n();
           ref="input"
           v-model:search-input="search"
           :model-value="selection"
+          :class="{
+            '[&_input:not([data-cy=edit-input])]:hidden': !!suggestionBeingEdited,
```

Comment:
I would rather not use the `[data-cy]` property in styling can we instead add a class and use that?